src/utilities/spacing/spacing.scss

```diff
@@ -17,11 +17,17 @@
     }
   }
 
+  /**
+    * 1. Ideally we could use the margin-inline shorthand, but it is not as well
+    *    supported in browsers.
+    */
   &-block-#{$name} {
     @if $property == margin {
-      margin-block: $value;
+      margin-block-end: $value; /* 1 */
+      margin-block-start: $value;
     } @else {
-      padding-block: $value;
+      padding-block-end: $value; /* 1 */
+      padding-block-start: $value;
```

Comment:
I don't think the number is necessary at all, since the comment is applying to the whole block of styles.